systemtests/src/main/java/com/github/streamshub/systemtests/setup/StrimziOperatorSetup.java

```diff
@@ -139,27 +120,37 @@ private ClusterRoleBinding[] getStrimziWatchAllCrbs() {
     }
 
     private ConfigMap getBundleConfigMap() {
-        return new ConfigMapBuilder(TestFrameUtils.configFromYaml(strimziResources.stream()
-            .filter(file -> file.getName().startsWith("050-ConfigMap")).toList().get(0), ConfigMap.class))
-            .editMetadata()
-                .withNamespace(this.deploymentNamespace)
-                .withName(this.deploymentName)
-            .endMetadata()
-            .build();
+        return strimziResources.stream()
+            .filter(ConfigMap.class::isInstance)
+            .map(ConfigMap.class::cast)
+            .map(r -> new ConfigMapBuilder(r)
+                .editMetadata()
+                    .withNamespace(this.deploymentNamespace)
+                    .withName(this.deploymentName)
+                .endMetadata()
+                .build())
+            .findFirst()
+            .orElseThrow();
```

Comment:
Not required, but you could cut down some of this, if you want, with a common get resource method. E.g.
```java
private <T> T getResource(Class<T> type) {
    return strimziResources.stream()
        .filter(type::isInstance)
        .map(type::cast)
        .findFirst()
        .orElseThrow(() -> new NoSuchElementException("No resource of type " + type));
}
```
Then all the other methods become simply the builder:
```java
private ServiceAccount getBundleServiceAccount() {
    return new ServiceAccountBuilder(getResource(ServiceAccount.class))
        .editMetadata()
            .withNamespace(this.deploymentNamespace)
        .endMetadata()
        .build();
}
```

Reply:
Sounds great. Thanks